packages/bitcore-wallet-service/src/lib/expressapp.ts

```diff
@@ -1129,6 +1129,28 @@ export class ExpressApp {
       });
     });
 
+    router.get('/v1/nonce/:address', (req, res) => {
```

Comment:
This request will take a bit, and it is not cached and it is open to everyone. I suggest we do getAuthServer, so at least it has to be attached to a wallet.

Reply:
good point. It is definitely better to use getAuthServer.